.changes/next-release/bugfix-4dc2263c-25fa-4d07-a6e1-69389f5ec001.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "fix wrongly formatted supplemental context for inline suggestion"
```

Comment:
```suggestion
  "description" : "Fix poor inline suggestions from Amazon Q caused by improperly formatted supplemental context"
```
?

Reply:
sgtm thanks richard. i believe i will always need your help looking at my wording thanks to my

Reply:
poor english ^__^